misc/i18n/generateTranslations.ts

```diff
@@ -1,28 +1,26 @@
+//@ts-ignore
 import { writeFile } from "fs"
+//@ts-ignore
 import { readFile } from "fs/promises"
 import yaml from "js-yaml"
 import { TsUtils } from '../../src/lib/TsUtils'
 import { betaInformation } from "./betaInformation"
 import pathConfig from './AvatarBaseType.json'
 import AvatarConfig from './AvatarConfig.json'
-import rankConfig from './AvatarRankConfig.json'
-import skillConfig from './AvatarSkillConfig.json'
 import damageConfig from './DamageType.json'
 import lightconeConfig from './EquipmentConfig.json'
-import lightconeRankConfig from './EquipmentSkillConfig.json'
 import relicSetConfig from './RelicSetConfig.json'
 import relicEffectConfig from './RelicSetSkillConfig.json'
 
 const precisionRound = TsUtils.precisionRound
 
-const Locales = ['zh', 'de', 'en', 'es', 'fr', 'id', 'ja', 'ko', 'pt', 'ru', 'th', 'vi']
+const inputLocales = ['zh', 'de', 'en', 'es', 'fr', 'id', 'ja', 'ko', 'pt', 'ru', 'th', 'vi'] as const
 
-const TrailblazerPaths = ['Warrior', 'Knight', 'Shaman']
+const outputLocales = [...inputLocales, 'it'] as const
```

Comment:
Im noticing like 3 different naming schemes for constants in this file which we should probably try to keep consistent throughout the codebase: `outputLocales` `TB_NAMES` `Overrides`. IMO prefer camelCase since these are local variables/not exported?